CHANGELOG.rst

```diff
@@ -11,15 +11,20 @@ Known bugs:
 Breaking changes:
 
 New features and notable changes:
+
 - Accept `NAN %` which is used in GCOV 7.5.0 instead of an invalid value. (:issue:`651`)
 - New :option:`--json-base` to define a base bath used in JSON reports. (:issue:`656`)
+- New nox session `bundle_app` to generate a portable application with pyinstaller. (:issue:`661`)
```

Comment:
Could link to the cookbook entry here.

Also, remember that RST needs double backticks for code markup :)

Reply:
🙈